lib_expense/src/test/java/com/example/expenselogger/lib_expense/domain/fakes/FakeExpenseContract.kt

```diff
@@ -7,14 +7,19 @@ public class FakeExpenseContract : ExpenseContract {
 
     private val database = mutableListOf<Expense>()
 
-    override suspend fun saveExpense(expense: Expense) {
+    override suspend fun insertExpense(expense: Expense): Long {
```

Comment:
You can in fact delete this fake implementation and write the unit test with mocks

Reply:
Okay, I'll do that. Thank you